maisim/maisim.Game.Tests/Visual/ComponentV2/TestSceneBeatmapCard.cs

```diff
@@ -0,0 +1,74 @@
+using System.Reflection;
+using maisim.Game.Beatmaps;
+using maisim.Game.Graphics.UserInterfaceV2;
+using maisim.Game.Utils;
+using NUnit.Framework;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace maisim.Game.Tests.Visual.ComponentV2;
+
+public class TestSceneBeatmapCard : maisimTestScene
+{
+    private Bindable<DifficultyLevel> difficultyLevel = new Bindable<DifficultyLevel>();
+
+    private Bindable<BeatmapSet> beatmapSet =  new Bindable<BeatmapSet>(new BeatmapSetTestFixture().BeatmapSet);
+
+    private void difficultyLevelChanged(ValueChangedEvent<DifficultyLevel> difficultyLevelEvent) => updateBeatmapInfo();
+    private void beatmapSetChanged(ValueChangedEvent<BeatmapSet> beatmapSetEvent) => updateBeatmapInfo();
+
+    private string beatmapList;
+    private TextFlowContainer beatmapListContainer;
+
+    public TestSceneBeatmapCard()
+    {
+        Children = new Drawable[]
+        {
+            new BeatmapSetInfoBox(difficultyLevel, beatmapSet)
+            {
+                Anchor = Anchor.TopRight,
+                Origin = Anchor.TopRight,
+                RelativeSizeAxes = Axes.Y,
+            },
+            beatmapListContainer = new TextFlowContainer()
+            {
+                Anchor = Anchor.BottomCentre,
+                Origin = Anchor.Centre,
+                RelativeSizeAxes = Axes.Both,
+                Text = ""
+            }
+        };
+        difficultyLevel.BindValueChanged(difficultyLevelChanged, true);
+        beatmapSet.BindValueChanged(beatmapSetChanged, true);
```

Comment:
you can inline these into lambda instead of using methods for one-liners:
```suggestion
        beatmapSet.BindValueChanged(_ => updateBeatmapInfo(), true);
```